src/Microsoft.DotNet.ProjectModel/ProjectContext.cs

```diff
@@ -22,6 +23,8 @@ public class ProjectContext
 
         public Project ProjectFile => RootProject.Project;
 
+        public LockFile LockFile { get; }
```

Comment:
We don't have to have this property. The lock file can be found under the project directory. `Path.Combine(ProjectDirectory, LockFile.FileName)`.